tfhe-zk-pok/src/proofs/mod.rs

```diff
@@ -42,7 +75,9 @@ impl<T: ?Sized + IndexMut<usize>> IndexMut<usize> for OneBased<T> {
     }
 }
 
-#[derive(Clone, Debug, serde::Serialize, serde::Deserialize)]
+#[derive(
+    Clone, Debug, serde::Serialize, serde::Deserialize, CanonicalSerialize, CanonicalDeserialize,
+)]
```

Comment:
tfhe wrapper types might need this as well to be able to serialize without compression, as going through only the serialize/deserialize will re use the default serialization mode

Reply:
is the derive for serde traits required ?